### ISSUE TITLE
Data leakage issue

### ISSUE THREAD

**Issue by wx-song** [RIA-lab/OpHReda#1]
The performance improvement of this work is very significant, but I have a question to ask. In your augmented dataset, I noticed there are 102,995 data points, but the original training data for EpHod only had 7,124 data points. According to the method described in the original paper, if each training sample is augmented tenfold, the final count including the original training data should be 78,369 data points. Your augmented data clearly exceeds this theoretical value. May I ask if you included the entire EpHod dataset, including the test set, in the augmentation?
